### PR TITLE
Update normalize, add resize function, add enums InterpolationFlags, NormType

### PR DESCRIPTION
**Pull request by Charles-Schleich**
Update normalize more closely match opencv definition.
Add examples of using resize, normalize, cvt_color functions

Maybe we don't need individual executable to test each function we write, 
It would be nice if we could write some tests to effectively do some integration testing with Master of WasmEdge.
There are lots of places where issues can occur.

This commit is related to [WasmEdge#2795](https://github.com/WasmEdge/WasmEdge/pull/2795)

Also looking for discussion on library design / layout.

### CODE REVIEW

**Comment by Charles-Schleich** [comment]
@dannypsnl Could you give this a look, it doesn't look like I can add reviewers.

File: example/plugin-function-tests/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[package]
+name = "plugin-function-tests"
```
Comment (dannypsnl) [inline example/plugin-function-tests/Cargo.toml]: If it's a test, it should not in example; if it's in example, it should has "test" in name

**Review by dannypsnl — CHANGES_REQUESTED** [review]

File: example/plugin-function-tests/Cargo.toml (hunk 1 above)
Comment (Charles-Schleich) [reply example/plugin-function-tests/Cargo.toml]: Sure thing, updated.

File: example/plugin-function-examples/src/cvt_color.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+use std::fs;
+
+use opencvmini::{cvt_color, imdecode, imencode, ColorConversionCodes};
+
+fn main() {
+    let image = fs::read("./asset/colour-wheel.jpg").expect("failed to open image");
+    // let image = fs::read("./asset/color-wheel-additive.jpg").expect("failed to open image");
```
Comment (dannypsnl) [inline example/plugin-function-examples/src/cvt_color.rs]: legacy comment

**Review by dannypsnl — CHANGES_REQUESTED** [review]

File: example/plugin-function-examples/src/cvt_color.rs (hunk 2 above)
Comment (Charles-Schleich) [reply example/plugin-function-examples/src/cvt_color.rs]: Removed.

**Review by dannypsnl — APPROVED** [review]

**Pull request merged**